nxdrive/gui/api.py

```diff
@@ -443,6 +444,23 @@ def default_server_url_value(self) -> str:
         """Make daily job better for our developers :)"""
         return getenv("NXDRIVE_TEST_NUXEO_URL", "")
 
+    @pyqtSlot(str, result=bool)
+    def free_disk_space_under_limit(self, path: str) -> bool:
+        remaining_space = shutil.disk_usage(Path(path).parent).free
```

Comment:
You should leverage other methods. Here you could use `get_free_disk_space(path)` for instance.